### PR TITLE
Fix django51

### LINKED ISSUE #49: Django 5.1: ImportError: cannot import name 'get_storage_class' from 'django.core.files.storage'
When I tried to use `django-storages` with Django 5.1 I have got following error:

```
Traceback (most recent call last):
  File "/home/petr/soubory/programovani/blenderkit/blenderhub_server/manage.py", line 28, in <module>
    execute_from_command_line(sys.argv)
  File "/home/petr/.cache/pypoetry/virtualenvs/blenderkit-server-eqrfB9dY-py3.11/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/petr/.cache/pypoetry/virtualenvs/blenderkit-server-eqrfB9dY-py3.11/lib/python3.11/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/petr/.cache/pypoetry/virtualenvs/blenderkit-server-eqrfB9dY-py3.11/lib/python3.11/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/petr/.cache/pypoetry/virtualenvs/blenderkit-server-eqrfB9dY-py3.11/lib/python3.11/site-packages/django/apps/registry.py", line 124, in populate
    app_config.ready()
  File "/home/petr/.cache/pypoetry/virtualenvs/blenderkit-server-eqrfB9dY-py3.11/lib/python3.11/site-packages/wagtail_storages/apps.py", line 10, in ready
    from wagtail_storages import signal_handlers
  File "/home/petr/.cache/pypoetry/virtualenvs/blenderkit-server-eqrfB9dY-py3.11/lib/python3.11/site-packages/wagtail_storages/signal_handlers.py", line 9, in <module>
    from wagtail_storages.utils import (
  File "/home/petr/.cache/pypoetry/virtualenvs/blenderkit-server-eqrfB9dY-py3.11/lib/python3.11/site-packages/wagtail_storages/utils.py", line 4, in <module>
    from django.core.files.storage import get_storage_class
ImportError: cannot import name 'get_storage_class' from 'django.core.files.storage' (/home/petr/.cache/pypoetry/virtualenvs/blenderkit-server-eqrfB9dY-py3.11/lib/python3.11/site-packages/django/core/files/storage/__init__.py
```

### PR DESCRIPTION
**Pull request by PetrDlouhy**
- fix #49 (support for Django 5.1)
- drop support for Django <=3.2
- fix testing

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{38,39,310}-dj32-wagtail52
     py{38,39,310,311}-dj42-wagtail{52,60}
-    py{311,312}-dj50-wagtail{52,60}
+    py{311,312}-dj{50}-wagtail{52,60}
+    py{311,312}-dj{51}-wagtail{60}
```
Comment (zerolab) [inline tox.ini:5]: given 6.0 is no longer supported, I would say replace with 6.1 and 6.2

Comment (PetrDlouhy) [reply tox.ini:5]: @zerolab Thanks, I will do that. But I think it will be best if #48 would be merged first and then I will rebase above it.

Comment (zerolab) [reply tox.ini:5]: Fair call. I went ahead and merged it as it was approved. Can you rebase?

Comment (PetrDlouhy) [reply tox.ini]: @zerolab Done. Seems like all tests has passed.

**Comment by harryrhodes** [comment]
Hey is there any update on this issue at all? We are looking to implement this package into a project running Django 51 and are facing the same issue as the one linked. Any update would be appreciated, thanks!

**Comment by PetrDlouhy** [comment]
From my point of view it is finished, I am ready to make more changes if needed.

**Review by tm-kn — APPROVED** [review]
LGTM! Thank you for the contribution.

**Comment by jrthorne** [comment]
I just installed the below and am getting the below error. I must admit that the messaging here is rather terse. the fix linked above rererences commit
https://github.com/torchbox/wagtail-storages/commit/451fcbe45710601d0faafd75034c0153daa734c2
which does not exist on the repo for wagtail storages repository linked in the github issue above:
https://github.com/torchbox/wagtail-storages/pull/50
which says the solution is:
https://github.com/torchbox/wagtail-storages/commit/451fcbe45710601d0faafd75034c0153daa734c2
which is an update to a KVPair in a YML file.
=== ERROR
  File "/usr/local/lib/python3.12/site-packages/django/template/backends/django.py", line 151, in get_template_tag_modules
    for name in get_package_libraries(pkg):
  File "/usr/local/lib/python3.12/site-packages/django/template/backends/django.py", line 176, in get_package_libraries
    raise InvalidTemplateLibrary(
django.template.library.InvalidTemplateLibrary: Invalid template library specified. ImportError raised when trying to load 'sass_processor.templatetags.sass_tags': cannot import name 'get_storage_class' from 'django.core.files.storage' (/usr/local/lib/python3.12/site-packages/django/core/files/storage/__init__.py
==== BELOW requirements.txt
requests
Django==5.1
django-grappelli==3.0.3
django-session-security
wagtail==6.3
Pillow==9.2.0
mysqlclient==2.1.1
sentry-sdk==1.13.0
django-extensions==3.2.1
django-environ==0.9.0
django-dirtyfields==1.8.2
django-global-permissions==0.2.6
django-mathfilters==1.0.0
django-crispy-forms==1.14.0
django-select2==7.10.1
django-compressor==4.5.1
django-htmlmin==0.11.0
django-sass-processor==1.2.1
libsass==0.21.0
beautifulsoup4==4.11.1
django-bootstrap-datepicker-plus
ua-parser==0.16.1
django-user-agents==0.4.0
user-agents==2.2.0
djangorestframework

**Comment by zerolab** [comment]
Looking at the requirements.txt listed above, there is no wagtail-storages. And the error points to django-sass-processor

**Comment by tm-kn** [comment]
@jrthorne Looks like you might be on an old version of the django sass processor that does not support your Django version. Quick search throws this up: https://github.com/search?q=repo%3Ajrief%2Fdjango-sass-processor%20get_storage_class&type=code I think this is unrelated.

**Pull request merged**